### PR TITLE
sorter_to_h5 add to ugbio_core

### PR DESCRIPTION
**Pull request by avigailmo**
terra_pipelines pr: https://github.com/Ultimagen/terra_pipeline/pull/1465
VariantCallling pr: https://github.com/Ultimagen/VariantCalling/pull/232

### CODE REVIEW

File: src/core/ugbio_core/convert_h5_to_json.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    args = ap.parse_args(argv[1:])
+    input_h5_filename = args.input_h5
+    output_json_filename = args.output_json
+    ignored_h5_key_substring = args.ignored_h5_key_substring
+    root_element = args.root_element
+
+    json_string = h5_utils.convert_h5_to_json(input_h5_filename, root_element, ignored_h5_key_substring)
+    with open(output_json_filename, "w", encoding="utf-8") as f:
+        f.write(json_string)
+        f.close()
```
Comment (inbalzelig-ugi) [inline src/core/ugbio_core/convert_h5_to_json.py:32]: you can delete this line

**Review by inbalzelig-ugi — APPROVED** [review]

File: src/core/tests/resources/026532-Lb_1866-Z0058-CATCTCAGTGCAATGAT.csv
[the remark this answers is not in the document]
Comment (avigailmo) [reply src/core/tests/resources/026532-Lb_1866-Z0058-CATCTCAGTGCAATGAT.csv:1]: I prefer to not change the namings and leave it as it was in VariantCalling

**Review by ilyasoifer — APPROVED** [review]
Looks good to me. Not sure about the directory structure, but maybe it is not crucial

**Comment by avigailmo** [comment]
> Looks good to me. Not sure about the directory structure, but maybe it is not crucial

let's talk about the directory structure one day and then I'll change accordingly

**Pull request merged**